src/controllers/accomodation.controller.js

```diff
@@ -105,6 +105,89 @@ class Accommodation {
     const query = await accomodationServices.bookAccommodation(req.user.id, req.body.accommodationId, req.body.roomId, req.body.departureDate, req.body.checkoutDate);
     return response.successMessage(res, 'Booking was successfully processed', 201, query);
   }
+
+  /**
+   * This function helps user to rate accommodation which he booked.
+   * @param {Object} req request
+   * @param {*} res response
+   * @returns {Object} return accommodation response
+   */
+  static async rateAccomodation(req, res) {
```

Comment:
rateAccommodation, Kindly check the word spelling. Update elsewhere it's applicable.

Reply:
I changed the variable name.